src/community/index.md.njk

```diff
@@ -1,52 +1,72 @@
 
-Reuse as much as possible and iterate based on user needs.
+### Propose a change to pages
+Anyone can suggest an improvement, report a bug or correct an error on our pages. Look for the ‘Help improve this page’ section at the end of each page to [propose a change using GitHub](https://design-system.service.gov.uk/community/propose-a-content-change-using-github/).
 
-Start by checking what exists in the [GOV.UK Design System](/).
+### Complete surveys and test releases
+We regularly ask our community to help guide our work or test new features before they’re released. To find out the next time we do, [sign up to our mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system).
 
-If something is not in the Design System, check the [community backlog](/community/backlog/) and [community resources and tools](/community/resources-and-tools/) to see what colleagues in other departments have done before.
+## Attend our events
 
-Reach out to the community to ask questions, gather examples and learn from others’ mistakes.
+Join one of our regular events where we share ideas and work together to solve common problems affecting service teams.
 
-### 2. Contribute back and help others
+### Design systems catchup
 
-The GOV.UK Design System is for everyone. Anyone can contribute evidenced-based changes.
+Chat with us every other Friday on Google Meet. It’s an informal chat with an agenda set each week by participants. [Add to calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=dGZpZ251Y2ttaHRkY3AwcTBxcHViMmY4ZmNfMjAyMjA5MDlUMTMwMDAwWiBjYWx2aW4ubGF1QGRpZ2l0YWwuY2FiaW5ldC1vZmZpY2UuZ292LnVr&tmsrc=calvin.lau%40digital.cabinet-office.gov.uk&scp=ALL) or [contact the team](https://design-system.service.gov.uk/get-in-touch/) for an invite.
 
-Think beyond your service and aim to design components and patterns that are scalable, reusable and can evolve over time.
+### Design System Day
+Every year, we host an online conference to collaborate and share knowledge about design systems with like-minded people, covering topics like accessibility, community and decision-making.
 
-Share research findings and examples with others. Be open to feedback or changes to your work.
+Registration for Design System Day 2022 on 15 and 16 November is now closed, but we’ll share videos, slides and notes to everyone after the event.
 
-Above all, be kind. Encourage others to contribute by being respectful when asking questions or giving feedback.
+## Join a workshop
+We organise workshops to gather ideas and feedback about work we’re doing, particularly when we know it will affect service teams. If you’d like to help the next time we do, [sign up to our mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system).
```

Comment:
```suggestion
We often organise workshops to gather ideas and feedback about work we’re doing, particularly when we know it will affect service teams. If you’d like to help the next time we do, [sign up to our mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system).
```